File: pset6/cash.py
from cs50 import get_float

# prompt valid input
amount = 0
while True:
    amount = get_float("Change owed: ")
    if amount > 0:
        break

# calculations
cents = round(amount * 100)
coins = cents // 25 + cents % 25 // 10 + cents % 25 % 10 // 5 + cents % 25 % 10 % 5

# output
print(coins)

File: pset1/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    // prompt valid intput
    float amount;
    do
    amount = get_float("Change owed: ");
    while (amount < 0);

    // calculations
    int cents = round(amount * 100);
    int coins = cents / 25 + cents % 25 / 10 + cents % 25 % 10 / 5 + cents % 25 % 10 % 5;

    // output
    printf("%i\n",coins);
}